azure/functions/eventhub.py

```diff
@@ -125,7 +125,7 @@ def decode_single_event(
             body=body,
             trigger_metadata=trigger_metadata,
             enqueued_time=cls._parse_datetime_metadata(
-                trigger_metadata, 'EnqueuedTime'),
```

Comment:
So are we making a change from `EnqueuedTime` to `EnqueuedTimeUTC`? Could that break existing users of the library which depend on that value to be in local time?

Reply:
Or was it never available in local time?

Reply:
> Or was it never available in local time?

It never works in local time. The current implementation will always return None to our customer.